prisma/schema.prisma

```diff
@@ -0,0 +1,29 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url = env("POSTGRES_CONNECTION_STRING")
+}
+
+model Trade {

```

Comment:
Flavor: In order to prevent the need to constantly reassign snake case to camel case, it's beneficial to use the same case you will be using in your code, as your database keys.